descriptions.md

```diff
@@ -459,6 +468,15 @@ Passes additional properties to a [`RestChannel`]{@link RestChannel} or [`Realti
 | params?: `Dict<String, String>` ||| TB2c | [Channel Parameters](https://ably.com/docs/realtime/channels/channel-parameters/overview) that configure the behavior of the channel. |
 | modes?: [ChannelMode] ||| TB2d | An array of [`ChannelMode`]{@link ChannelMode} objects. |
 
+
+## class DeriveOptions
+
+Passes additional properties to a [`RealtimeChannel`]{@link RealtimeChannel} object, such as filter, to create a new derived/synthetic channel.
```

Comment:
```suggestion
Passes properties to a [`RealtimeChannel`]{@link RealtimeChannel} object, such as a filter, to create a derived channel.
```

Suggestion to amend the wording slightly given it's mandatory rather than optional. Also agree with the associated [spec PR](https://github.com/ably/specification/pull/138#discussion_r1184835973) to avoid introducing 'synthetic' as well as 'derived'.